e2e/convert_docker_tarball/README.md

```diff
@@ -0,0 +1,9 @@
+# Convert a docker tarball as a base image
+
+In some cases, your legacy setup doesn't fetch a base image from a remote registry, instead you've produced your base image in a script and check or fetch the tarball.
+
+To generate it run;
+
+```bash
+docker buildx build e2e/convert_docker_tarball --output=type=docker,dest=e2e/convert_docker_tarball/image.tar --builder=cool_swirles -t test:latest
```

Comment:
nit: value for —builder has to be created with docker-container driver for buildx. We can remove this line.

Reply:
Maybe Add some doc for the bash script you created?